vault/hcp_link/capabilities/meta/meta.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```

Comment:
Just raising an idea here. We have a subdirectory for node status protobuf file. Would it make sense to move all protobuf files for other capabilities there too?